stix2generator/language/interop_custom.json

```diff
@@ -0,0 +1,309 @@
+        "type": "object",
+        "import": "common-properties",
+        "required": ["identities_class", "created_by_ref"],
+        "properties":{
+            "type": "grouping",
+            "created_by_ref": {
+                "type": "string",
+                "semantics": "stix-id",
+                "stix-type": "identity"
+            }
+        }
+    },
+    "incident": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["created_by_ref"],
+        "properties":{
+            "type": "incident",
+            "created_by_ref": {
+                "type": "string",
+                "semantics": "stix-id",
+                "stix-type": "identity"
+            }
+        }
+    },
+    "identity": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["identity_class", "created_by_ref"],
+        "properties":{
+            "type": "identity",
+            "created_by_ref": {
+                "type": "string",
+                "semantics": "stix-id",
+                "stix-type": "identity"
+            }
+        }
+    },
+    "indicator": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["indicator_types","name", "created_by_ref"],
+        "properties":{
+            "type": "indicator",
+            "created_by_ref": {
+                "type": "string",
+                "semantics": "stix-id",
+                "stix-type": "identity"
+            }
+        }
+    },
+    "infrastructure": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["infrastructure_types", "created_by_ref"],
+        "properties":{
+            "type": "infrastructure",
+            "created_by_ref": {
+                "type": "string",
+                "semantics": "stix-id",
+                "stix-type": "identity"
+            }
+        }
+    },
+    "intrusion-set": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["resource_level", "primary_motivations", "created_by_ref"],
+        "properties":{
+            "type": "intrusion set",
+            "created_by_ref": {
+                "type": "string",
+                "semantics": "stix-id",
+                "stix-type": "identity"
+            }
+        }
+    },
+    "location": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["region", "created_by_ref"],
+        "properties":{
+            "type": "location",
+            "created_by_ref": {
+                "type": "string",
+                "semantics": "stix-id",
+                "stix-type": "identity"
+            }
+        }
+    },
+    "malware": {
+        "type": "object",
+        "import": "common-properties",
+        "required": ["malware_types", "is_family", "first_seen", "last_seen", "architecture_execution_envs", "implementation_languages", "capabilities", "name", "created_by_ref"],
```

Comment:
`name` and `is_family` are already required by the standard registry.

Reply:
removed